src/common/transformations/src/transformations/common_optimizations/mul_conv_fusion.cpp

```diff
@@ -41,11 +71,12 @@ ov::pass::MultiplyConvolutionFusion::MultiplyConvolutionFusion() {
 
         const auto& weights_shape = weights.get_shape();
         const auto& mul_const_shape = mul_const.get_shape();
-        // Check if mul_const if broadcastable to weights.
+
```

Comment:
@mateusztabaka is also suggesting to add a following check here:
```
        if (input.get_partial_shape().size() < weights_shape.size())
            return false;
```
but I don't really understand what's the point of it here.

Reply:
Added